### ISSUE TITLE
unable to install veil  in Kali Linux "2017.3" x86_64

### ISSUE THREAD

**Issue by dcliss** [Veil-Framework/Veil#162]
## Veil version
3.0

## OS Used - all info (architecture, linux flavor, etc)

Kali Linux "2017.3" x86_64

## How did you install Veil? (Apt, Clone from Github, etc.)
git clone

## Did you run the setup script?
yes

## Pastebin link to error you are encountering (include console actions you took prior to error)

https://pastebin.com/Vjp7efmU
https://youtu.be/MRt72Pc1NDw
## Any additional info you want to tell me

I tried installing veil in a fresh kali vm with vmware tools installed , however it installs python and unable to go forward after pressing next . look at the video i uploaded to youtube.
inbtwn this is a fresh kali vm with no tools except vmware tools installed . https://youtu.be/MRt72Pc1NDw
 

thanks

**Comment by ChrisTruncer** [comment]
This is a duplicate issue. As I have stated in the previous issue, this is unfortunately something I cannot recreate. I run the setup script on my Kali vm, and it installs the whole way through. Have you tried the silent install which it asks at the beginning?

**Comment by ChrisTruncer** [comment]
I also downloaded on the latest release of Kali, and was successfully able to install Veil and its dependencies

**Comment by dcliss** [comment]
let me try again and get back to you in a hour and i will aso check on the silent install
